app/components/owned-card-list.cjsx

```diff
@@ -5,14 +5,20 @@ Translate = require 'react-translate-component'
 apiClient = require 'panoptes-client/lib/api-client'
 PromiseRenderer = require '../components/promise-renderer'
 OwnedCard = require '../partials/owned-card'
-{Link} = require 'react-router'
+{Link, State, Navigation} = require 'react-router'
```

Comment:
I'm pretty sure that following the changes in this PR, this file is no longer used on the project page - only on collections, favourites etc. In which case, I don't think this PR should be modifying it... Right?